mtags/src/main/scala-3/scala/meta/internal/pc/InferredTypeProvider.scala

```diff
@@ -236,7 +241,10 @@ final class InferredTypeProvider(
        */
       case Some(df @ DefDef(name, _, tpt, rhs)) =>
         def typeNameEdit =
-          val end = tpt.endPos.toLsp
+          val end =
```

Comment:
In scala 3.1.3, 
```scala
List((1,2)).map((a,b) => ...)
```
`a` and `b` are treated as methods, but their `tpt.SourcePos` == `df.namePos.startPos`, so we need this correctly insert type after name

Reply:
I just meant in the comment in the code :sweat_smile: